app/http.php

```diff
@@ -19,7 +19,7 @@
 
 $http = new Server("0.0.0.0", App::getEnv('PORT', 80));
 
-$payloadSize = max(4000000 /* 4mb */, App::getEnv('_APP_STORAGE_LIMIT', 10000000 /* 10mb */));
+$payloadSize = 6 * (1024 * 1024); // 5MB
```

Comment:
Can we mention the 1 MB buffer here?

Reply:
We don't need to, The whole payload size is 6 MB (i see i have different in comment). So for files we keep the chunk size to 5 MB so that total request doesn't exceed the payload size.

Reply:
I know, just explain it there so people know why the comment says 5 MB and is actually 6 MB 👍🏻